#std_msgs/Header header
# The information of uninteresting feature maps
float32[] feature
int32[] shape